app/src/main/java/com/mapbox/maps/testapp/examples/localization/LocalizationActivity.kt

```diff
@@ -0,0 +1,154 @@
+package com.mapbox.maps.testapp.examples.localization
+
+import android.os.Bundle
+import android.view.Menu
+import android.view.MenuItem
+import android.widget.Toast
+import androidx.appcompat.app.AppCompatActivity
+import com.mapbox.maps.MapboxMap
+import com.mapbox.maps.Style
+import com.mapbox.maps.extension.style.localization.localizeLabels
+import com.mapbox.maps.testapp.R
+import kotlinx.android.synthetic.main.activity_map_localization.*
+import kotlinx.android.synthetic.main.activity_map_overlay.mapView
+import java.util.*
+
+/**
+ * Example showcasing usage of localization.
```

Comment:
I think we can be a bit more thorough in giving the reader context on what and how:

> Example showcasing how to localize a map client side to a specific locale using `Style#localizeLabels(locale: Locale?)`. This function will attempt to localize a map into a selected locale if the symbol layers are using a Mapbox source and the locale is being provided as part of the vector source data. This feature supports both v7 and v8 of Mapbox style spec version.